hbw/app/models/hbw/common/adapter.rb

```diff
@@ -76,7 +76,13 @@ def drop_processes(entity_code, entity_class)
       end
 
       def entity_tasks(email, entity_code, entity_class, size = 1000)
-        task_list_wrapped(email, entity_code, entity_class, size)
+        tasks = task_list_wrapped(email, entity_code, entity_class, size)
+
+        tasks.map do |task|
```

Comment:
`each` or `tap`
Also you can create method to call it with `tasks.tap(&:fetch_form)`